autotest/test_order3_hf_commond_chat.py

```diff
@@ -0,0 +1,150 @@
+import allure
+import pytest
+import yaml
+from utils.run_client_chat import hfCommondLineTest
+
+
+@pytest.fixture(scope='class', autouse=True)
+def case_config(request):
```

Comment:
I believe  this fixture is used  in most of the test cases and it should not be changed during its life time. So why not make it to the global fixture and only define and create once.